docs/configuring-playbook.md

```diff
@@ -20,7 +20,8 @@ You can then follow these steps inside the playbook directory:
 
 
 For a basic Matrix installation, that's all you need.
-For a more custom setup, see the [Other configuration options](#other-configuration-options) below.
+For a more custom setup, see the [Other configuration options](#other-configuration-options) below.  
```

Comment:
Linking to `matrix-base`'s `defaults/main.yml` is of limited use. There are only a handful of things there, while the tens of other roles contain hundreds of other variables.